js/form-state.js

```diff
@@ -0,0 +1,25 @@
+const adForm = document.querySelector('.ad-form');
+const mapFilters = document.querySelector('.map__filters');
+
+
+const addFormState = (elementFormAds, elementFormFilters) => {
+  elementFormAds.classList.add('ad-form--disabled');
```

Comment:
it is uniq function, so you don't need params, you can just define nodes right here, inside function